### DISCUSSION TITLE
A deas

### DISCUSSION THREAD

**Discussion by longzho** [T8RIN/ImageToolbox#1496] [Ideas]
May I inquire if you would consider this feature?![Screenshot_2024-11-25-19-35-04-73_df198e732186825c8df26e3c5a10d7cd.jpg](https://github.com/user-attachments/assets/87a07a33-4ffd-4c43-85d5-24660bb4aa08)

In some cases, this is a very practical feature.Thanks!❤️❤️❤️

**Comment by T8RIN** [comment]
It's already available

**Reply by longzho** [reply]
I'm sorry, I made a bit of a mistake in the expression.The current batch editing function for images does not seem to allow synchronous cropping of images of the same size. It can only edit image one by one at a time in sequence, and cropping is not possible.https://github.com/user-attachments/assets/c9485b3d-d0af-4a0b-a30c-66de79090205

**Reply by longzho** [reply]
Synchronize cropping, not synchronous scaling features.

**Reply by T8RIN** [reply]
You can use crop resize mode
